### ISSUE TITLE
Last Channel button

### ISSUE THREAD

**Issue by r33ms** [whmcs-smarters/IPTV-Smarters-TV-Demo-APK#43]
Currently, Dpad left and Dpad right have no functionality while watching a channel fullscreen.  Can we set DPAD Left to tune to the previous channel that was watched?  This would be a great feature to flip between two channels when watching things like sports.

**Comment by wendellbalous** [comment]
I agree

**Comment by AZBoxEmperador** [comment]
I think it´ll be better if we set Dpad down to show the last 5 or 6 channels watched...

**Comment by r33ms** [comment]
If they could somehow make a window slide out from the side showing the last 5 or 6 channels, yeah, that would be awesome.  I mentioned dpad left because right now up and down changes the channel.  Dpad left to popout a menu from either side would be nice, good suggestion!

**Comment by r33ms** [comment]
Aman, any updates or acknowledgement on this?  Thanks.

**Comment by tomtruitt** [comment]
Agreed

**Comment by ghasul** [comment]
I agree, Channel Return (Last Channel) is a feature that I'd like to see implemented.
It's a quick way to flip back and forth between sports events or news shows.

**Comment by joebrookhill1** [comment]
Any update on this?
I see it was opened in Apr 18 but nothing since Jul 19.
I used PP for a while and it had this feature and was really useful. 
It is really cumbersome to scan and then view a channel knowing to go back to last channel is going to be a long process if last channel is in another group.
